js/service.js

```diff
@@ -51,8 +51,10 @@ function getDataFromURL(route, params = {}, token = {}) {
   };
 
   Object.assign(params, defaultParams);
-
   return new Promise((resolve, reject) => {
+    if(params.help_type) {
+      params.help_type = params.help_type.split(`-`).join(` `).split(`and`).join(`&`);
```

Comment:
If the word "and" is in the middle of an actual word, this fails. I added a help type to staging to demonstrate this- spin up a local server pointed at the staging API, and visit http://localhost:3000/entry/379. There's a help type "Stand together" and the link fails because under the hood it's searching for "St& together"

Reply:
Yeah that's true. Let me try and find a suitable fix for that :+1: